pyoffers/models/affiliate.py

```diff
@@ -0,0 +1,85 @@
+# coding: utf-8
+from .core import Model, ModelManager, RelatedManager
+
+AFFILIATE_USER_PARAMS = (
+    'access',
+    'cell_phone',
+    'email',
+    'first_name',
+    'last_name',
+    'password',
+    'password_confirmation',
+    'phone',
+    'status',
+    'title',
+    'wants_alerts'
+)
+
+
+class AffiliateUser(Model):
+    generic_methods = ('update', 'delete')
+
+
+class AffiliateUserManager(ModelManager):
+    model = AffiliateUser
+    name = 'affiliate_users'
+    generic_methods = (
+        'create',
+        'update',
+        'delete',
+        'find_by_id',
+        'find_all',
+        'find_all_ids',
+    )
+
+
+class RelatedUserManager(RelatedManager):
+    related_object_name = 'affiliate_id'
+
+    def get(self):
+        try:
+            return self.base_manager.find_all(**{self.related_object_name: self.id})[0]
+        except IndexError:
+            return None
+
+
+class Affiliate(Model):
+    """
+    An Affiliate.
+    """
+    generic_methods = ('update', 'delete')
+
+    @property
+    def affiliate_user(self):
```

Comment:
Please, rename ``affiliate_user`` -> ``user``. I think, that this repetition will only decrease readability